packages/sanity/src/core/field/diff/components/DiffTooltip.tsx

```diff
@@ -75,26 +76,35 @@ function AnnotationItem({annotation}: {annotation: AnnotationDetails}) {
   const {t} = useTranslation()
 
   return (
-    <Inline space={2}>
-      <Flex
-        align="center"
-        paddingRight={3}
-        style={{
-          backgroundColor: color.background,
-          color: color.text,
-          borderRadius: 'calc(23px / 2)',
-        }}
-      >
-        <UserAvatar user={author} />
-        <Inline paddingLeft={2}>
-          <Text muted size={1} style={{color: color.text}}>
-            {user ? user.displayName : t('changes.loading-author')}
+    <>
+      {annotation.event ? (
+        <>
+          <Card borderBottom marginBottom={2} />
+          <Event event={annotation.event} showChangesBy="inline" />
+        </>
+      ) : (
+        <Inline space={2}>
+          <Flex
+            align="center"
+            paddingRight={3}
+            style={{
+              backgroundColor: color.background,
+              color: color.text,
+              borderRadius: 'calc(23px / 2)',
```

Comment:
specific™️

Reply:
100% 
This should probably not be like that... but this was an existing change, so trying not to mess with old components

Reply:
Oh yeah for sure, I just found it funny 😄

Reply:
`23px/2` ought to be enough for anybody!

Reply:
🤝